dotcom-rendering/src/web/components/ShowHideButton.tsx

```diff
@@ -22,7 +22,7 @@ const showHideButtonCss = (
 	text-decoration: none;
 
 	${from.wide} {
-		position: absolute;
+		position: relative;
```

Comment:
Absolute position moves the button out of view; relative position brings it back to where it's meant to be: 
![image](https://user-images.githubusercontent.com/102960844/218802242-5a0f1966-8f9e-4eb8-b83e-144c547dbd37.png)

![image](https://user-images.githubusercontent.com/102960844/218802340-66b52b6d-808c-4da6-9445-671cbee1daf5.png)

Reply:
Looking at the original PR, I think [we could remove almost all of the styles](https://github.com/guardian/dotcom-rendering/commit/f22a22b4b2d910b7ed566b670fa7da8b0ebc6bbd#diff-d9a8d15e0990e7c3f8c2f36863a67ea118ce050b7d014757cbdda6ec80f1d251)

```tsx
```

Reply:
I still think we can remove a lot more styles!